packages/protocol/contracts/common/Accounts.sol

```diff
@@ -123,7 +123,10 @@ contract Accounts is
 
   /**
    * @notice Returns the storage, major, minor, and patch version of the contract.
-   * @return The storage, major, minor, and patch version of the contract.
+   * @return storage Storage version of the contract.
+   * @return major Major version of the contract.
+   * @return minor Minor version of the contract.
+   * @return patch Patch version of the contract.
```

Comment:
Since these are not named return values, can omit the first words in each of these lines.

Reply:
Thanks, updated